### PR TITLE
feat:query validation added in api-gateway

### LINKED ISSUE #39: Validate for empty query object
Now, if you pass empty query it will give a weird error. Query needs to be validated.

### PR DESCRIPTION
**Pull request by manojown**
resolve #39

Query validation added in Api-gateway.

### CODE REVIEW

File: packages/cubejs-api-gateway/index.js [hunk 1]
```diff
@@ -195,7 +195,7 @@ class ApiGateway {
   initApp(app) {
     app.get(`${this.basePath}/v1/load`, this.checkAuthMiddleware, (async (req, res) => {
       try {
-        const query = JSON.parse(req.query.query);
+        const query = await Joi.validate(JSON.parse(req.query.query),querySchema)
```
Comment (paveltiunov) [inline packages/cubejs-api-gateway/index.js]: I believe this one is redundant as it called by `normalizeQuery`?

**Comment by paveltiunov** [comment]
@manojown Hey Manoj! Everything looks great except one line. Left a comment there.

File: packages/cubejs-api-gateway/index.js (hunk 1 above)
Comment (manojown) [reply packages/cubejs-api-gateway/index.js]: ya right . Thanks

**Comment by paveltiunov** [comment]
@manojown 🙌

**Pull request merged**